addons/hr_timesheet/views/project_views.xml

```diff
@@ -287,5 +287,21 @@
             </field>
         </record>
 
+        <record id="project_task_view_pivot" model="ir.ui.view">
+            <field name="name">project.task.view.pivot.inherited</field>
+            <field name="model">project.task</field>
+            <field name="inherit_id" ref="project.view_project_task_pivot"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='stage_id']" position='after'>
+                    <field name="planned_hours" widget="timesheet_uom"/>
+                    <field name="remaining_hours" widget="timesheet_uom"/>
+                    <field name="effective_hours" widget="timesheet_uom"/>
+                    <field name="total_hours_spent" widget="timesheet_uom"/>
+                    <field name="overtime" widget="timesheet_uom"/>
+                    <field name="subtask_effective_hours" widget="timesheet_uom"/>
```

Comment:
For readability and comprehension to directly know those fields are measures and not `type="row"` 🙂 
```suggestion
                    <field name="planned_hours" widget="timesheet_uom" type="measure"/>
                    <field name="remaining_hours" widget="timesheet_uom" type="measure"/>
                    <field name="effective_hours" widget="timesheet_uom" type="measure"/>
                    <field name="total_hours_spent" widget="timesheet_uom" type="measure"/>
                    <field name="overtime" widget="timesheet_uom" type="measure"/>
                    <field name="subtask_effective_hours" widget="timesheet_uom" type="measure"/>
```